arm-servicefabric/2016-09-01/swagger/servicefabric.json

```diff
@@ -404,43 +404,43 @@
       }
     },
     "EndpointRangeDescription": {
-      "description": "Endpoint range description",
+      "description": "Port range details",
       "properties": {
         "startPort": {
-          "description": "Endpoint start port",
+          "description": "Starting port of a range of ports",
           "type": "integer"
         },
         "endPort": {
-          "description": "Endpoint end port",
+          "description": "End port of a range of ports",
           "type": "integer"
         }
       }
     },
     "NodeTypeDescription": {
-      "description": "NodeType description",
+      "description": "NodeType details",
       "properties": {
         "name": {
-          "description": "NodeType description name",
+          "description": "Name of NodeType",
           "type": "string"
         },
         "placementProperties": {
-          "description": "Placement properties key",
+          "description": "Placement property key",
           "type": "object",
           "additionalProperties": {
-            "description": "Placement properties value",
+            "description": "Placement property value",
             "type": "string"
           }
         },
         "capacities": {
-          "description": "Capacities key",
+          "description": "Capacity property key",
```

Comment:
Provide better description e.g. "The capacity tags applied to the nodes in the node type, the cluster resource manager uses these tags to understand how much of a resource a node has"